### PR TITLE
[FEEDBACK NEEDED.]Sinful multi belly mob.

### PR DESCRIPTION
**Pull request by lbnesquik**
This adds the framework for the 'sarlaac'. Basically supposed to be exactly what it sounds like. The interesting part is that it's a mob with a whooping 2 stomachs. Wow. Much vore. :dog: 
It needs review (not necessarily by coders) about it's implementation. It also ,of course, need sprites and some descriptions. 
Can some staff lad give pretty colors to PR plz thanks.

- [ ] Get a new name so we don't get sued. (Probably wouldn't happen but i dunno.)
- [X] Get sprites
- [ ] Get good descriptions for insides and outside
- [X] Remove my shame.
- [ ] Figure how to make this *actually immobile*. It should never move. Sometimes it move. Especially with bump vore.
- [ ] *Maaaaybe* add more levels.
- [ ] **Difficult** Make it so the mob ITSELF calls the transfer.

### CODE REVIEW

**Comment by lbnesquik** [comment]
https://github.com/VOREStation/VOREStation/issues/509

**Comment by lbnesquik** [comment]
Wooh it passed checks.

File: code/modules/mob/living/simple_animal/vore/sarlaac.dm [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+/mob/living/simple_animal/retaliate/sarlaac
+	name = "Sarlacc"	//Need a new name so no suing.
+	desc = "Some sort of massive maw sticking out of the ground. Seems safe." //Placeholder plz suggest better.
+	//tt_desc = "Semaeostomeae virginus" //No idea what name it should have
+	icon = 'icons/mob/vore64x64.dmi'//Placeholder.
+	icon_dead = "deathclaw-dead"
+	icon_living = "deathclaw"
+	icon_state = "deathclaw"
+
```
[the remark this answers is not in the document]
Comment (lbnesquik) [reply code/modules/mob/living/simple_animal/vore/sarlaac.dm]: That probably can work.

**Comment by lbnesquik** [comment]
I've ~~stolen~~ found some pretty good sprites from a russian artist. I'll be trying to get them in.

File: code/modules/vore/eating/simple_animal_vr.dm [hunk 2]
```diff
@@ -19,6 +19,9 @@
 		to_chat(src,"<span class='warning'>Put your hands to good use instead!</span>")
 		return
 	feed_grabbed_to_self(src,T)
+	
+	if(vore_animation)//This affects all bolt action and shotguns. 
+		flick("[icon_state]-eating", src)//This plays any pumping
```
Comment (Arokha) [inline code/modules/vore/eating/simple_animal_vr.dm]: I suggest abstracting this further, into `if(vore_animation) handle_eating_anim()` and just have the generic flick there as the prototype. Then if a mob wants to do silly things like have 7 different animations for different methods of vore or whatever then that's entirely doable.

File: code/modules/mob/living/simple_animal/vore/sarlaac.dm [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+
+// Make sure you don't call ..() on this one, otherwise you duplicate work.
+/mob/living/simple_animal/init_vore()
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/vore/sarlaac.dm]: But... this gives EVERY s_a two bellies. Don't do that. :v

Comment (lbnesquik) [reply code/modules/mob/living/simple_animal/vore/sarlaac.dm]: Confirmed with a catgirl. I'll fix.

**Comment by Arokha** [comment]
Seems like it'd work.

**Comment by SpadesNeil** [comment]
i have no idea

**Pull request closed without merging**